### ISSUE TITLE
NTR: Dihydrouracil high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8469]
New term request Dihydrouracil high in CSF (elthat_111121061407): Cerebrospinal fluid  UMLS: 1847704

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000211
- label: Elevated CSF dihydrouracil concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: PMID:6467612
- def: The concentration of dihydrouracil in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8469
- synonyms: Dihydrouracil high in CSF(NONE)
